tox.ini

```diff
@@ -78,6 +81,15 @@ commands =
     notudfdoctest: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} and not udf" {posargs:} tests
     local: {env:SNOWFLAKE_PYTEST_CMD} --local_testing_mode -m "(integ and localtest) or unit or mock_unit" {posargs:} tests
     dailynotdoctest: {env:SNOWFLAKE_PYTEST_DAILY_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} or udf" {posargs:} tests
+    # Snowpark pandas commands:
+    snowparkpandasnotdoctest: {env:SNOWFLAKE_PYTEST_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} tests/unit/modin tests/integ/modin
```

Comment:
SNOWFLAKE_PYTEST_CMD has SNOWFLAKE_PYTEST_IGNORE_MODIN_CMD, and here we just run tests on  `tests/unit/modin` and `tests/integ/modin`, so I guess actually they are not running?